Sources/Shadow/Shadow.swift

```diff
@@ -0,0 +1,106 @@
+import SwiftUI
+
+public enum Shadow: String, CaseIterable {
+    case small, medium, large, xLarge
+    
+    var opacity1: Double {
+        switch self {
+        case .small:
+            return 0.16
+        case .medium:
+            return 0.16
+        case .large:
+            return 0.1
+        case .xLarge:
+            return 0.2
+        }
+    }
+    var opacity2: Double {
+        switch self {
+        case .small:
+            return 0.12
+        case .medium:
+            return 0.1
+        case .large:
+            return 0.2
+        case .xLarge:
+            return 0.1
+        }
+    }
+    var radius1: CGFloat {
+        switch self {
+        case .small:
+            return 0.5
+        case .medium:
+            return 4
+        case .large:
+            return 10
+        case .xLarge:
+            return 14
+        }
+    }
+    var radius2: CGFloat {
+        switch self {
+        case .small:
+            return 3
+        case .medium:
+            return 3
+        case .large:
+            return 6
+        case .xLarge:
+            return 9
+        }
+    }
+    var x1: CGFloat {
+        0
+    }
+    var x2: CGFloat {
+        0
+    }
+    var y1: CGFloat {
+        switch self {
+        case .small:
+            return 1
+        case .medium:
+            return 3
+        case .large:
+            return 10
+        case .xLarge:
+            return 14
+        }
+    }
+    var y2: CGFloat {
+        switch self {
+        case .small:
+            return 1
+        case .medium:
+            return 3
+        case .large:
+            return 6
+        case .xLarge:
+            return 9
+        }
+    }
+}
+
+struct ShadowType: ViewModifier {
```

Comment:
If this should be expose to user,
Mark it as public,
But if this should not be exposed, then mark it as private in order to hide detail implementation from outside of this file.

Also rename it to something more meaningful,
Proposed: ShadowViewModifier